lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -3465,7 +3551,8 @@ def show_actions(mod) # :nodoc:
 
   def show_advanced_options(mod) # :nodoc:
     mod_opt = Serializer::ReadableText.dump_advanced_options(mod, '   ')
-    print("\nModule advanced options:\n\n#{mod_opt}\n") if (mod_opt and mod_opt.length > 0)
+    print("\nModule options (#{mod.fullname}):\n\n#{mod_opt}\n") if (mod_opt and mod_opt.length > 0)
```

Comment:
Was this line added intentionally? It seems incorrect, `mod_opt` is advanced options here.